src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +29,35 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #
+              {todo.id}
+
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={close}
```

Comment:
```suggestion
            <button
              type="button"
              className="delete"
              data-cy="modal-close"
              onClick={close}
              aria-label="delete"
```
aria label attribute can fix such eslint issue